src/baseevents.cpp

```diff
@@ -74,6 +74,7 @@ bool BaseEvents::loadFromXml()
 
 		if (!success || !registerEvent(event, node)) {
 			delete event;
+			return false;
```

Comment:
So @ranisalt @djarek what should we do?
As of right now they don't return anything = you will think it reloaded without errors.

Should I add a boolean that will be returned as Makalo said?